deploy/crds/openebs_v1alpha1_blockdeviceclaim_cr.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: openebs.io/v1alpha1
-kind: DeviceRequest
+kind: BlockDeviceClaim
 metadata:
-  name: test-devicerequest
+  name: test-blockdeviceclaim
 spec:
   capacity: 300
```

Comment:
can this spec follow the same mechanism like PVC and resource requests?

Reply:
Let me check that @kmova